src/codeflare_sdk.egg-info/SOURCES.txt

```diff
@@ -7,17 +7,19 @@ src/codeflare_sdk.egg-info/SOURCES.txt
 src/codeflare_sdk.egg-info/dependency_links.txt
 src/codeflare_sdk.egg-info/top_level.txt
 src/codeflare_sdk/cluster/__init__.py
-src/codeflare_sdk/cluster/auth.py
 src/codeflare_sdk/cluster/awload.py
 src/codeflare_sdk/cluster/cluster.py
 src/codeflare_sdk/cluster/config.py
 src/codeflare_sdk/cluster/model.py
 src/codeflare_sdk/cluster/widgets.py
+src/codeflare_sdk/common/__init__.py
+src/codeflare_sdk/common/kubernetes_cluster/__init__.py
+src/codeflare_sdk/common/kubernetes_cluster/auth.py
+src/codeflare_sdk/common/kubernetes_cluster/kube_api_helpers.py
 src/codeflare_sdk/job/__init__.py
 src/codeflare_sdk/job/ray_jobs.py
 src/codeflare_sdk/utils/__init__.py
 src/codeflare_sdk/utils/demos.py
 src/codeflare_sdk/utils/generate_cert.py
 src/codeflare_sdk/utils/generate_yaml.py
-src/codeflare_sdk/utils/kube_api_helpers.py
```

Comment:
nit: for this file to be ignored in `.gitignore`, it needs to be deleted.
No need to hold merge for this.